### DISCUSSION TITLE
ShowColumnStripes x ShowRowStripes type

### DISCUSSION THREAD

**Discussion by montovaneli** [qax-os/excelize#1579] [Q&A]
Hello! Sorry if it's a silly question, but why is the `ShowRowStripes` parameter of an `excelize.Table` of type `*bool`, while the `ShowColumnStripes `parameter is of type `bool`?

**Comment by xuri** [comment]
The row stripes were enabled by default when you add a table with the `AddTable` function, even if you did not specify the `ShowRowStripes` filed value in the `excelize.Table`. Because the default value of the boolean data type was false in the Go language, we used pointer data type to represent the user not set value for this field and made this field value as true by default, so we using the `*bool` data type for distinguish unset or set as false. The column stripes were disabled by default, so we can define the data type of the `ShowColumnStripes` field as boolean.
